src/app/containers/StoryPromo/index.jsx

```diff
@@ -81,7 +103,15 @@ const StoryPromo = ({ item, lazyLoadImage, topStory }) => {
       {headline && (
         <Headline script={script} service={service} topStory={topStory}>
           <Link href={url}>
-            <LinkContents item={item} />
+            {isLive ? (
+              <LiveComponent
+                service={service}
+                linkcontents={headline}
```

Comment:
Howcome this is `linkcontents`? Imo `headline` would be better as it doesnt confuse with `linkcontents` which is a JSX node